packages/c/codex-icons.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "codex-icons",
+  "filename": "codex-icons.umd.js",
+  "description": "icons of the Codex design system for Wikimedia",
+  "keywords": [
+    "codex",
+    "design-system",
+    "components",
+    "front-end"
+  ],
+  "autoupdate": {
+    "source": "npm",
+    "target": "@wikimedia/codex-icons",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "*.@(js|mjs|json|svg)"
```

```suggestion
          "*.@(js|mjs|json)",
          "images/*.svg"
```
It looks like the actual icons are in a sub-directory?

---

Good catch, thanks – I guess I wasn’t sure if the blobs are directory-recursive or not.